packages/flutter_tools/test/general.shard/forbidden_imports_test.dart

```diff
@@ -136,18 +138,20 @@ void main() {
 
   test('no unauthorized imports of package:file/local.dart', () {
     final List<String> allowedPath = <String>[
-      globals.fs.path.join(flutterTools, 'lib', 'src', 'base', 'file_system.dart'),
+      fileSystem.path.join(flutterTools, 'test', 'general.shard', 'forbidden_imports_test.dart'),
+      fileSystem.path.join(flutterTools, 'test', 'integration.shard', 'test_utils.dart'),
```

Comment:
would it be possible to allow the entire `//test/integration.shard/` dir?

Reply:
we could - this sort of enforces using the existing test_utils, which could be helpful if we do want to change the file system in the future

Reply:
oh yeah, makes sense